packages/lib-user/src/components/UserHome/components/Dashboard/components/StatsTabs/StatsTabs.js

```diff
@@ -55,13 +55,23 @@ function Stat({ stats }) {
 }
 
 export default function StatsTabs({ statsPreview }) {
+  const [index, setIndex] = useState(1)
```

Comment:
Would have preferred to not add state to the component, but I think it's the only way to have the second tab as the initially active tab per https://v2.grommet.io/tabs#activeIndex ?

Reply:
No problem. Would rather work with Grommet's expectation of state rather than have to refactor this component.